cmd/telemetrygen/internal/metrics/worker.go

```diff
@@ -28,7 +29,13 @@ type worker struct {
 
 func (w worker) simulateMetrics(res *resource.Resource, exporter sdkmetric.Exporter) {
```

Comment:
Instead of generating three metrics I would suggest having a CLI option that allows you to specify the metric type.

To keep things simple, let's do a first PR for adding support for adding sums first and we can tackle histograms in a separate PR.

Reply:
Sounds good. I will split up the PRs today

Reply:
Added new CLI parameter `metric-type`

<img width="856" alt="image" src="https://github.com/open-telemetry/opentelemetry-collector-contrib/assets/558256/a7414b27-1c79-4202-9570-471a827d9c28">